generator/vogtle_data_generator.py

```diff
@@ -76,7 +66,7 @@ def generate_inspections(self, rows):
                  "effort_required|facility|targeted_flag|target_amt\n"
 
         with open('{}inspections.csv'
```

Comment:
Python now supports fstrings.  They are a little faster and I think a little easier to read.

```python
with open(f'{self.directory}inspections.csv', 'w+') as output_file:
```

The only place I've seen they can cause problems is with logging, which is optimized for the old % style strings.